.env.sample

```diff
@@ -0,0 +1,7 @@
+DATABASE_URL=postgres://user:pwd@localhost:5433/postgres
+GOOGLE_CALLBACK_URL=http://localhost:5000/auth/google/callback
+GOOGLE_CLIENT_SECRET=your-client-secret
+GOOGLE_CLIENT_ID=your-client-id
+FACEBOOK_CLIENT_SECRET=your-client-secret
+FACEBOOK_CLIENT_ID=your-client-id
+FACEBOOK_CALLBACK_URL=http://localhost:5000/auth/facebook/callback
```

Comment:
I'm assuming http when testing local, and https on the actual deployed heroku app callback.